charts/apisix/values.yaml

```diff
@@ -440,14 +440,20 @@ apisix:
 
       stream: |
 
-    # -- Add custom [lua_shared_dict](https://github.com/openresty/lua-nginx-module#toc88) settings,
+    # -- Add custom [lua_shared_dict](https://github.com/openresty/lua-nginx-module?tab=readme-ov-file#lua_shared_dict) settings,
     # click [here](https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix/values.yaml#L27-L30) to learn the format of a shared dict
     customLuaSharedDicts: []
       # - name: foo
       #   size: 10k
       # - name: bar
       #   size: 1m
 
+    # -- Override default [lua_shared_dict](https://github.com/apache/apisix/blob/master/conf/config.yaml.example#L250-L276) settings,
+    # click [here](https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix/values.yaml#L27-L30) to learn the format of a shared dict
+    luaSharedDicts: []
+      # - name: prometheus-metrics
+      #   size: 20m
```

Comment:
This example is inappropriate, prometheus is defined using the special command for `meta { lua_shared_dict }`, which cannot be overridden by `http { lua_shared_dict }`, but it does override other dict values, such as those used by limit-count.

ref: https://github.com/apache/apisix/blob/master/conf/config.yaml.example#L165-L168

This is a special case for creating shared memory between the HTTP subsystem and the stream subsystem.

Reply:
@bzp2010 I actually tried `meta` but it doesn't work. And overriding it under `http` does

At least it didn't work for me. Maybe I'm missing something.

Reply:
@dennispan, can you provide your test steps? Maybe we can find the problem from them

Reply:
I used this to test (after a re-deployment to pick up the configuration changes):

```
/usr/local/openresty/nginx/sbin/nginx -T \
  -p /usr/local/apisix/ \
  -c conf/nginx.conf \
| grep lua_shared_dict
```

Reply:
@Baoyuantop @bzp2010 any update on this? Anything else needed from me to move this forward?